packages/frontend-api/src/Resources/config/graphql-types/ProductFilter/ParameterFilterOptionDecorator.types.yaml

```
@@ -0,0 +1,10 @@
+ParameterFilterOptionDecorator:
+    type: input-object
+    decorator: true
+    config:
+        description: "Parameter filter option"
+        fields:
+            parameter:
+                type: "Parameter!"
+            values:
+                type: "[ParameterValueFilterOption]!"
```

```suggestion
                type: "[ParameterValueFilterOption!]!"
```

so it's not possible to return null as one item in an array (empty array is still allowed)

---

190f141abae92afe3f2fc09f1f542853f4c79247